### PR TITLE
Implementing the Posts component

### PR DESCRIPTION
**Pull request by OmerSchechter**
#236

screen shot 2016-12-26 at 21 36 07

### CODE REVIEW

File: client/src/app/services/posts.service.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { Injectable } from '@angular/core';
-
```
Comment (nirgn975) [inline client/src/app/services/posts.service.ts]: Revert this change. There should be a space between "their" (the angular team) and "our" imports ([link to the style guide](https://angular.io/styleguide#!#03-06)).

Comment (OmerSchechter) [inline client/src/app/services/posts.service.ts]: Done.
